lib/matplotlib/offsetbox.py

```diff
@@ -1205,6 +1225,7 @@ def __init__(self, arr,
                  **kwargs
                  ):
 
+        OffsetBox.__init__(self)
```

Comment:
I found most of these issues due to examples failing to build.  I do not recall seeing anything in the `__init__` methods that are doing subtle things like expecting the child classes to have pre-set some value, but silently use a default value if they have not.

My suspicion is that the current ordering was not by design.